sales_page/js/app.js

```diff
@@ -156,7 +155,6 @@ salesForm.addEventListener('submit', function(event) {
 });
 
 //TODO: Strech goal
```

Comment:
I really like how you add theTODOs. It really helps you organize your code and figure out how to hit all MVP!